hack/generator/pkg/astmodel/resource.go

```diff
@@ -226,22 +226,22 @@ func (definition *ResourceType) WithOwner(owner *TypeName) *ResourceType {
 	return &result
 }
 
-// RequiredImports returns a list of packages required by this
-func (definition *ResourceType) RequiredImports() []PackageReference {
-	typeImports := definition.spec.RequiredImports()
+// RequiredPackageReferences returns a list of packages required by this
+func (definition *ResourceType) RequiredPackageReferences() []PackageReference {
+	references := definition.spec.RequiredPackageReferences()
 
 	if definition.status != nil {
-		typeImports = append(typeImports, definition.status.RequiredImports()...)
+		references = append(references, definition.status.RequiredPackageReferences()...)
 	}
 
-	typeImports = append(typeImports, MetaV1PackageReference)
-	typeImports = append(typeImports, MakeGenRuntimePackageReference())
-	typeImports = append(typeImports, MakeExternalPackageReference("fmt"))
+	references = append(references, MetaV1PackageReference)
```

Comment:
random observation (don't take this as something you should fix...): It's sorta wonky we have a hardcoded reference for `metav1PackageReference`, and we could for `GenRuntime` too but we don't. Just once of those really small inconsistencies.

Reply:
Something to fix later, perhaps.